test/regress.c

```diff
@@ -146,8 +146,7 @@ test(char const * path, int limit)
     pkt_additional = NULL;
     nestegg_packet_additional_data(pkt, 1, &pkt_additional, &pkt_additional_length);
 
-    printf("%d", pkt_keyframe_status);
-    printf(" %u %llu %u", pkt_track, (unsigned long long) pkt_tstamp, pkt_cnt);
+    printf("%d %u %llu %u", pkt_keyframe, pkt_track, (unsigned long long) pkt_tstamp, pkt_cnt);
```

Comment:
Switch the order of track and keyframe please, it's a bit easier to parse the .ok files if the track number is the first thing listed.